### ISSUE TITLE
Retrode 2 - Second SNES controller port not working

### ISSUE THREAD

**Issue by oivivio** [OpenEmu/OpenEmu#1863]
Hi!

I cannot seem to get my second SNES controller mapped to Player 2 in the Openemu controller settings.  Retrode 2 is registering controller 2 communication via the red light on the front of the Retrode 2. 

The first SNES controller has no problem mapping to Player 1.  Is anyone else having this issues?

I'm currently running the latest Openemu 1.0.4 on OS X 10.10.2. 

Thanks,

Brad

**Comment by clobber** [comment]
@oivivio You do not need to spam other open issues. That doesn't help and just adds noise. You already have an open issue so someone will get to it when they get to it.

**Comment by oivivio** [comment]
@clobber Apologies.  I have posted my issue over two weeks ago without any response.  I did not realize I was "spamming" another thread when including feedback on an issue similar to mine.

**Comment by clobber** [comment]
Indeed. No, can't conclude it's a similar issue. You are using an adapter (for which only one of us on the team has) and those are always problematic. We also automap controls for that device and maybe only one port is recognized by the application. So, you have a **rare** adapter which hardly any of us can test with and that might answer why you haven't gotten a response to "is anyone else having this issue" for a number of weeks.

**Comment by oivivio** [comment]
Ok, thank you clobber.  I will await response from you or the team when available.

**Comment by andybrito** [comment]
I bought a Retrode 2 and a Mega Drive 3 button controller without know that they were not compatible with OpenEmu, i am very sad... is there any chance of the next version be at least compatible with 1 Mega Drive controller ?
